kotlin/.buildscript/configure-compose.gradle

```diff
@@ -19,6 +19,7 @@ android {
     compose true
   }
   composeOptions {
+    kotlinCompilerVersion "1.3.61-dev-withExperimentalGoogleExtensions-20200129"
```

Comment:
Just curious - why is this required?

Reply:
The docs said to add it, but they were wrong about other things – I just removed it and it seems to work fine.